shopping-cart-service/src/main/resources/application.properties

```diff
@@ -1,3 +1,5 @@
 spring.application.name=shopping-cart
 
-springdoc.packagesToScan=agh.fis.shopping_cart.controller
\ No newline at end of file
+springdoc.packagesToScan=agh.fis.shopping_cart.controller
+
+server.error.include-message=always
```

Comment:
Out of curiosity, what's that flag for?

Reply:
Without this flag message returned by ResponseStatusException is empty.